xtask/src/custom_workload.rs

```diff
@@ -61,11 +82,19 @@ pub fn new_custom_workload(
     })
 }
 
-fn rand_key(id: u64) -> Vec<u8> {
+fn rand_key() -> [u8; 32] {
     // keys must be uniformly distributed
     use rand::{RngCore as _, SeedableRng as _};
-    let mut seed = [0; 16];
-    seed[0..8].copy_from_slice(&id.to_le_bytes());
+    use std::time::{SystemTime, UNIX_EPOCH};
+    let mut seed: [u8; 16] = match SystemTime::now().duration_since(UNIX_EPOCH) {
```

Comment:
it seems cleaner to pass an `&mut impl Rng` into the function rather than re-seeding it each time.